Dockerfile

```diff
@@ -51,7 +50,8 @@ ARG BASE_YUM_REPO=release
 LABEL maintainer "OSG Software <help@opensciencegrid.org>"
 LABEL name "osg-ce-condor"
 
-RUN yum install -y osg-ce-condor && \
+RUN yum install -y osg-ce-condor \
+                   htcondor-ce-view && \
```

Comment:
We don't need it here either, let's rip it out completely

Reply:
Do we know that? We run the hosted CEs so we know what we want there, but the osg-ce-condor image is being used by non-OSG staff...

Reply:
Who? AFAIK, the Open Science CE is the only service that uses this

Reply:
OK